### PR TITLE
Use Jade

### PR DESCRIPTION
**Pull request by dsturley**

### CODE REVIEW

File: data/panel/panel.js [hunk 1]
```diff
@@ -13,204 +13,201 @@
...
+		$id('actions').querySelector('button').focus();
+	}
+
+	function displayNodeDetails(nodeNumber) {
+		var node = rule.nodes[nodeNumber];
+		var impact = $id('impact');
+		$id('currentNode').textContent = nodeNumber + 1;
+		impact.textContent = node.impact.charAt(0).toUpperCase() + node.impact.slice(1);
+		impact.className = node.impact;
+		$id('issue-details').innerHTML = JST.details({
```
Comment (dsturley) [inline data/panel/panel.js]: `innerHTML` the Jade details templates

File: data/panel/panel.js [hunk 2]
```diff
@@ -13,204 +13,201 @@
...
+				return;
+			}
+		}, false);
+	}
+
+	function refresh(showMsg) {
+		details.classList.add('empty');
+		list.innerHTML = '';
+		status.innerHTML = '';
+		$id('issue-details').innerHTML = '';
```
Comment (dsturley) [inline data/panel/panel.js]: `innerHTML = ''` to empty.

File: data/panel/panel.js [hunk 3]
```diff
@@ -13,204 +13,201 @@
...
+							any: node.any,
+							all: node.all.concat(node.none)
+						};
+					})
+				};
 			}, 0);
-      status.innerHTML = total + ' violations found.';
-			list.innerHTML = compiledListTemplate({ violations: violations });
+			status.textContent = total + ' violations found.';
+			list.innerHTML = JST.sidebar({
```
Comment (dsturley) [inline data/panel/panel.js]: `innerHTML` Jade sidebar template

File: data/panel/panel.js [hunk 4]
```diff
@@ -13,204 +13,201 @@
...
+				violations: violations
+			});
 		} else {
-      details.classList.add('no-violations');
-      status.innerHTML = 'No violations found.';
-			$id('issue-details').innerHTML = '<p class="axe-message">Congratulations! No accessibility violations found. Now you should perform manual testing using assistive technologies like NVDA, VoiceOver and JAWS.</p>';
+			details.classList.add('no-violations');
+			status.textContent = 'No violations found.';
+			$id('issue-details').innerHTML = JST.congrats();
```
Comment (dsturley) [inline data/panel/panel.js]: `innerHTML` Jade congrats template

**Comment by dylanb** [comment]
Did you forget to commit the templates.js file?

**Pull request merged**